### DISCUSSION TITLE
General project comments

### DISCUSSION THREAD

**Discussion by gug903** [BobbyCasas/cs3443-app-prog-group12-pathfinders-maze#27] [General]
Most of the project is complete from what I understand we need to translate the hint ArrayList into numerical data for rooms and have the UML diagram

**Comment by gug903** [comment]
I think the guy whos git hub id im waiting on could do the uml if no one has done it yet

**Comment by EthanHammonsRed** [comment]
hello I think I am here I can do the uml if no one has done it yet

**Reply by gug903** [reply]
alright

**Comment by EthanHammonsRed** [comment]
Hello, I finished the uml I think it is correct but I will edit it if the project changes or you see problems with it
![Maze_uml](https://user-images.githubusercontent.com/93632909/143391781-b89d121f-3afe-4ab2-8ae6-522dae930f34.png)
.

**Reply by BobbyCasas** [reply]
Hi, your variables and methods all look great except maze was meant to be private on MazeController and a few more variables and methods were added to MazeController, Maze, and Room. GetHints was deleted. And for the relationships, it looks like all of them are incorrect. These would be the correct relationships.
![image](https://user-images.githubusercontent.com/45516996/143674976-448437be-b2f7-4b9d-be8c-82b42e64fb7e.png)

**Reply by BobbyCasas** [reply]
Also, "mazeSolution" was changed to "mazeRooms" in Maze and MazeController, and MazeController's increaseMinotaurChance() returns a boolean now. And Maze.fxml shouldn't have any variables just like the other fxml files.

**Comment by EthanHammonsRed** [comment]
also it will not let me upload the raw violet file here.

**Reply by gug903** [reply]
did u select upload file on add file and if fo then you can export it as a image from uml or screenshot it then upload it

**Comment by BobbyCasas** [comment]
@MrWhiskersss Have you finished the UML yet? We gotta turn in the project today.

**Reply by EthanHammonsRed** [reply]
I'm editing it right now although you probably already did it yourself.

**Comment by EthanHammonsRed** [comment]
![Maze_final_uml](https://user-images.githubusercontent.com/93632909/143814693-faa8fa0f-9a60-47cd-aa70-e092c4ab0c2e.png)

**Comment by EthanHammonsRed** [comment]
sorry I hope you didn't have to do it yourself :(

**Comment by Iconnick** [comment]
![Group12ProjectUML](https://user-images.githubusercontent.com/91359425/143814836-6d35f5b5-f08f-4773-959f-42b85f1f7b4e.png)

**Reply by EthanHammonsRed** [reply]
nice!

**Reply by Iconnick** [reply]
finished just when you did. Oh well lol.

**Comment by Iconnick** [comment]
![Group12Uml](https://user-images.githubusercontent.com/91359425/143815375-1f0da83d-4e83-4195-9bec-d3dc57638df1.png)
